src/data/config.py

```diff
@@ -81,7 +80,22 @@ def __init__(self, filename):
             if 'optional' in crash_value and 'split_columns' in crash_value['optional']:
                 self.split_columns += crash_value['optional']['split_columns'].keys()
 
-        
+    def __get_feature_type(self, feat: dict) -> str:
```

Comment:
can we do just a single underscore at the beginning of the function name?

Reply:
Will fix, for some reason I thought that was syntax for hidden functions.